Documentation/ContinuousIntegration.md

```diff
@@ -0,0 +1,145 @@
+This page describes the Continuous Integration (CI) used for BabylonNative.
+The CI helps improve the quality of code for all platforms and avoid regression by testing the engine against various test scenes. Idealy, there is one test scene for every single feature of babylonjs.
+Validation works by running the engine with a JS script that creates a scene. The rendering is captured and compared with a reference image.
+If the rendered and reference images are too different, the test fails, an error will raise in the CI (failed build) and rendered/error images are stored as an download artefact than can help debug.
+This is done for various platforms when a PR is opened, for nightly build and when a PR is merged in master branch.
+CI will also take care of code quality: Warnings are treated as error and will break the build. A PR can't be merged if the build fails.
+
+CI validation is composed of 3 main parts:
+- The test datas
+- The ValidationTests app
+- The platform integration
+
+# Test datas
+They are shared with babylonjs for the most part.
+Validation test scenes are listed in ```Apps\ValidationTests\Scripts\config.json```. It lists playground identifier that will be opened sequentialy. Per PG is may have additional parameters for validation pixels difference threshold.
+This json is a subset of babylonjs' one. The goal is to use the exact same file once the engine is in a sufficiently stable state.
+
+```
```

Comment:
Specify the content type to improve syntax highlighting in some markdown renderers.
```suggestion
```json
```